Gopkg.lock

```diff
@@ -1,6 +1,20 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
```

Comment:
If @mcardosos is okay with this change it's fine with me. In general let's be deliberate about changes to Gopkg.lock since it effects everything in this repo.

Reply:
Well, the lock file changes a lot. It changes every time we do `dep ensure` (if there happens to be new commits on our dependencies, which happens often)

Reply:
okay, so no problem here. i wonder if there are good practice recommendations for lock files?